src/database/models/accommodation.js

```diff
@@ -0,0 +1,37 @@
+export default (sequelize, DataTypes) => {
+  const accommodation = sequelize.define('accommodation', {
+    userId: DataTypes.INTEGER,
+    myImage: DataTypes.STRING,
+    accommodationName: DataTypes.STRING,
+    accommodationAddress: DataTypes.STRING,
+    accommodationDescription: DataTypes.TEXT,
+    cost: DataTypes.STRING,
```

Comment:
can make this property integer, and create the `currency` property to hold the currency, so in the response, the cost property will be a combination of cost and currency, i.e const cost =  ` ${cost} ${currency}`

Reply:
Yes I will work on it soon